libbpf-tools/biosnoop.bpf.c

```diff
@@ -4,13 +4,16 @@
 #include <bpf/bpf_helpers.h>
 #include <bpf/bpf_core_read.h>
 #include <bpf/bpf_tracing.h>
+#include <linux/version.h>
```

Comment:
it's misleading to include linux/version.h, because it also defines LINUX_VERSION_CODE, which is wrong

You can either copy/paste KERNEL_VERSION macro, or wait a bit for latest libbpf sync, which will happen today/tomorrow and will bring KERNEL_VERSION macro pre-defined in bpf_helpers.h

Reply:
Thanks for the insight. I'll revise this PR.